source/plugins/go/src/telemetry.go

```diff
@@ -68,6 +68,11 @@ var (
 	PromMonitorPodsLabelSelectorLength int
 	//Tracks the number of monitor kubernetes pods field selectors and sent only from prometheus sidecar (uses ContainerLogTelemetryTicker)
 	PromMonitorPodsFieldSelectorLength int
```

Comment:
as part of this change, can we make following changes 

1. from DockerVersion to "cri" 
2.  workspaceID to WSID
3. workspaceCloud to "WSCloud"
4. PodName to Pod

Reply:
Not changing the DockerVersion field. We use the same field name in the actual KubeNodeInventory table. I think better to have this consistency. What do you think @ganga1980 ?

Reply:
> Not changing the DockerVersion field. We use the same field name in the actual KubeNodeInventory table. I think better to have this consistency. Even for PodName, it is used in ContainerInventory. What do you think @ganga1980 ?

> @jatakiajanvi12 , since this is telemetry and we dont need to keep consistency. as long as we undestand these fields  we are good.  We kept the DockerVersion in KubeNodeInventory  table to avoid breaking changes for customers.

Reply:
Updated the fields